aten/src/ATen/native/native_functions.yaml

```diff
@@ -6625,6 +6625,24 @@
     CPU: legacy::cpu::_th_histc
     CUDA: _histc_cuda
 
+- func: histogram.bins_tensor_out(Tensor self, Tensor bins, Scalar min=0, Scalar max=0, Tensor? weight=None, bool density=False, *, Tensor(a!) hist, Tensor(b!) bin_edges) -> (Tensor(a!) hist, Tensor(b!) bin_edges)
+  dispatch:
+    CPU: histogram_out_cpu
+
+- func: histogram.bins_tensor(Tensor self, Tensor bins, Scalar min=0, Scalar max=0, Tensor? weight=None, bool density=False) -> (Tensor hist, Tensor bin_edges)
```

Comment:
Would you elaborate on min/max vs range?

Which, if either, is more consistent with PyTorch's current operators?

Is being divergent worth the cost of requiring NumPy users to look up that it's min/max vs range?

If we use min/max here, would that been that translations of other NumPy operators will require a similar change? Or do they use min/max? Or are there not many examples of NumPy ops that specify ranges?

Reply:
In torch we have histc, clip/clamp, and hardtanh taking min/max style parameters.
numpy has clip and char.count taking min/max.
numpy.histogram looks to be the only operator in either library accepting a range tuple.

Overall, I think taking a range tuple in torch.histogram should be OK; clip/clamp and hardtanh don't seem closely related enough to warrant trying to be consistent with them.

Reply:
Per offline discussion, the plan here is to accept "min"/"max" keyword arguments in the official documentation but also secretly support a "range" keyword argument for numpy-friendliness.